src/main/java/io/debezium/connector/cassandra/SchemaHolder.java

```diff
@@ -31,49 +37,64 @@
  * by {@link SchemaProcessor} periodically.
  */
 public class SchemaHolder {
-
     private static final String NAMESPACE = "io.debezium.connector.cassandra";
     private static final Logger LOGGER = LoggerFactory.getLogger(SchemaHolder.class);
 
-    private final Map<KeyspaceTable, KeyValueSchema> tableToKVSchemaMap = new ConcurrentHashMap<>();
+    public final Map<KeyspaceTable, KeyValueSchema> tableToKVSchemaMap = new ConcurrentHashMap<>();
+    public final Multimap<String, String> keyspaceTableMap = Multimaps.synchronizedListMultimap(ArrayListMultimap.create());
```

Comment:
What is this `keyspaceTableMap` used for here?

Reply:
Hi can you explain what this `keyspaceTableMap` is used for?

Reply:
It was a remaining of the old solution where I was holding some other map paralel to this one because if I wanted to ask if some key is there, I had to construct a new object every time. If you ask if some key is present in tableToKVSchemaMap, you have to do something like 

````
if (tableToKVSchemaMap.contains(new KeyspaceTable(...)) {
   ...
}
````

and creating an object here is just no too much performance friendly if it is going to be asked frequently. Having a paralel structure based just on a <String,String> is way way easier.

But I do not think we need that now, do we ... it is not used anywhere (but it was previously and I just forgot to remove it).